web-next/app/copilot/agent/page.module.css

```
@@ -0,0 +1,1426 @@
+
+.root .codeLang {
+  color: #bc8cff;
+  font-weight: 700;
+  font-size: 11px;
+  letter-spacing: 0.07em;
+  text-transform: uppercase;
+}
+
+.root .codeBody {
+  padding: 20px 22px;
+  font-family: "Cascadia Code", "Fira Code", "Consolas", monospace;
```

Stylelint (17.9.0)

[error] Expected no quotes around "Consolas" (font-family-name-quotes)

(font-family-name-quotes)